apps/android/boot.js

```diff
@@ -197,6 +197,13 @@
           lastSteps = steps;
         }, event.int*1000);
       },
+      "env": function() {
+        gbSend({
+          t: "env",
+          env: process.env,
```

Comment:
If we are going to do this on every connection to gadgetbridge maybe we want to only send the info of process.env we know we want over to GB? To lessen the activity of bluetooth data transfer which I think draws more energy than manipulating the data on the bangle? Or maybe this is negligible?

Reply:
On the other hand we also want to keep bootcode as short as possible. I don't know :p

Reply:
That is a good point. In the line below, it reads setting.json on every connection as well - not sure if desired, or if we have a simpler way of determining if the bangle is programmable or not.

Reply:
I guess we could do some kind of hack over on GB side? that tries to send over code to execute that will send an answer back if programmable is toggled on. If no response is given GB can assume that programmable is toggled off.

Reply:
> On the other hand we also want to keep bootcode as short as possible. I don't know :p

A tangent regarding this, would it help speed up the boot process if we moved much of boot files logic into libraries?

Reply:
> would it help speed up the boot process if we moved much of boot files logic into libraries?

If the code was code that gets called at boot time, it'd actually make it slower. It'd only help if it were code that only gets called occasionally, as then you can push the parsing overhead to the point when the code is first run.

Reply:
Reduced this to fw, hw and repl.